internal/proto/query_coord.proto

```diff
@@ -215,16 +215,15 @@ message WatchDmChannelsRequest {
   int64 collectionID = 3;
   repeated int64 partitionIDs = 4;
   repeated data.VchannelInfo infos = 5;
-  schema.CollectionSchema schema = 6;
+  schema.CollectionSchema schema = 6; // keep it for compatibility of rolling upgrade from 2.2.x to 2.3
   repeated data.SegmentInfo exclude_infos = 7;
-  LoadMetaInfo load_meta = 8;
+  LoadMetaInfo load_meta = 8; // keep it for compatibility of rolling upgrade from 2.2.x to 2.3
   int64 replicaID = 9;
   map<int64, data.SegmentInfo> segment_infos = 10;
   // Deprecated
   // for node down load balance, need to remove offline node in time after every watchDmChannel finish.
   int64 offlineNodeID = 11;
   int64 version = 12;
-  repeated index.IndexInfo index_info_list = 13;
 }
```

Comment:
Add some comments to remind others that serial number 11 has already been used.

Reply:
it's ok because index_info_list is added into master branch and has never been released.